### ISSUE TITLE
Filter lists evaluation

### ISSUE THREAD

**Issue by jspenguin2017** [NanoAdblocker/NanoCore#321]
Use this megathread for discussion about addition and removal of filter lists in the assets manifest.

**Comment by jspenguin2017** [comment]
Addition: `https://github.com/realodix/AdBlockID`

This filter list seems to be better maintained than the current regional filter.

**Comment by jspenguin2017** [comment]
Removal: `https://github.com/NanoMeow/MDLMirror`

It is been well over a year since anything was added.

**Comment by jspenguin2017** [comment]
Removal: `Nano filters – Whitelist`

It is not used for a long time.

**Comment by llacb47** [comment]
> Addition: `https://github.com/realodix/AdBlockID`
> 
> This filter list seems to be better maintained than the current regional filter.

I believe this list is better for uBo as it has scriptlet support, unlike ABPindo.

https://github.com/NanoMeow/QuickReports/issues/3918

**Comment by jspenguin2017** [comment]
@llacb47 Yes, I realize that, but more I diverge from uBO harder it is to piggyback on uAssets.
